### ISSUE TITLE
Numbers in namespace seems not supported.

### ISSUE THREAD

**Issue by kriebb** [pengweiqhca/Xunit.DependencyInjection#49]
**Describe the bug**
When a namespace starts with a number, .net & visual studio supports this, by adding a _. However, the assemblyname is supported to set this without a _.

You also can't set an assembly:StartupType .... that would've solved my problem also.

E.g.

AssemblyName = HelloWorld.01
RootNameSpace = HelloWorld._01

**To Reproduce**
1. Create a library, and use after a . an number.
E.g. HelloWorld.01
2. Create a Startup class, that Xunit.DependencyInjection will resolve.
3. Create somesort of class and register. Inject that class in your test.
4. Run the test.
5. TestRunner will give you an exception that the fixture is not found.

**Expected behavior**
- For each . in a namespace, the first character should be checked in order to set an _ if that is found.
- Ensure we have the possibility to give our own class by using some sort of startuptype as attribute.

**Comment by pengweiqhca** [comment]
Yes, you can declare [XunitStartupFullName](https://github.com/pengweiqhca/Xunit.DependencyInjection#how-to-find-startup) in project file.
